packages/react-flow-editor/src/Editor/helpers/DnD/helpers.ts

```diff
@@ -0,0 +1,8 @@
+import { DragItemState } from "@/Editor/state"
+import { DragItemType } from "@/Editor/types"
+import { Node } from "@/types"
+
+export const isNodesHaveStateToReset = (nodes: Node[], dragItem: DragItemState) =>
```

Comment:
You can get value from `DragItemAtom` throw `DragItemAtom.get()`, instead of pass it as argument to function. It should work